app/assets/stylesheets/application/base/_elements.scss

```diff
@@ -72,5 +72,9 @@ mark {
 
 hr {
   border: 0;
-  border-bottom: 1px solid $light-gray-color;
+  border-top: 1px solid $light-gray-color;
+  -webkit-margin-before: 0;
```

Comment:
Properties should be ordered border, border-top, -webkit-margin-after, -webkit-margin-before, -webkit-margin-end, -webkit-margin-start